core-bundle/src/Resources/contao/pages/PageRegular.php

```diff
@@ -825,24 +825,15 @@ protected function createFooterScripts($objLayout, $objPage = null)
 
 			$meta = array
 			(
-				'@context' => 'https://contao.org/',
-				'@type' => 'PageMetaData',
+				'@context' => 'https://schema.contao.org/',
+				'@type' => 'RegularPage',
 				'pageId' => (int) $objPage->id,
-				'language' => $objPage->language,
-				'title' => $objPage->pageTitle ?: $objPage->title,
 				'noSearch' => $noSearch,
 				'protected' => (bool) $objPage->protected,
 				'groups' => array_map('intval', array_filter((array) $objPage->groups)),
 				'fePreview' => System::getContainer()->get('contao.security.token_checker')->isPreviewMode()
 			);
 
-			$token = System::getContainer()->get('security.token_storage')->getToken();
-
-			if ($token !== null && $token->getUser() instanceof FrontendUser)
-			{
-				$meta['memberId'] = (int) $token->getUser()->id;
```

Comment:
Do we not need the member ID anymore?

Reply:
We never did, that was just a „maybe we can use it one day“ :)

Reply:
I see. 😄